docs/UserGuide.md

```diff
@@ -919,13 +918,13 @@ Other examples:
 
 #### Finding an attendance record: `find-attendance`
 
-If you need to find out a specific student's attendance for a specific week, make use of this command.
+If you need to find out a specific student's attendance for a specific week, you can make use of this command.
 
 **Format**: `find-attendance c\CLASS_INDEX l\LESSON_INDEX s\STUDENT_INDEX w\WEEK`
 * Finds the attendance record at the specified `CLASS_INDEX` + `LESSON_INDEX` + `STUDENT_INDEX` + `WEEK`.
 
 For example:
-1. It is the end of a semester and it's time to key in your students' attendance into luminus. You enter the
+1. It is the end of the semester and you wish to key in your students' attendance into luminus. You enter the
```

Comment:
Not sure if we should be including the reference to LumiNUS, perhaps it should be included in the glossary then?
```suggestion
1. It is the end of the semester and you wish to key in your students' attendance into LumiNUS. You enter the
```

Reply:
isnt our target users nus cs tutors tho... wouldn't luminus be something the users are familiar with..

Reply:
Its better to play safe and include it since we did also include `Telegram` in the glossary.